util/travis_build.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# if docker is installed - call make within the qmk docker image
+if docker -v &> /dev/null; then
+    function make() {
+      docker run --rm -w /qmk_firmware/ -v "$(pwd)":/qmk_firmware qmkfm/qmk_firmware make "$@"
```

Comment:
```suggestion
      docker run --rm -w /qmk_firmware/ -v "$PWD":/qmk_firmware qmkfm/qmk_firmware make "$@"
```
It's safe to use `$PWD` here.

Reply:
Done